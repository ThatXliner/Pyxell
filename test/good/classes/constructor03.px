
class C def
    Int x
    String s

    constructor(Int x, String s: ".") def
        self.x = x
        self.s = s

c = C(8)
print c.x, c.s

c = C(s="abc", x=-3)
print c.x, c.s
