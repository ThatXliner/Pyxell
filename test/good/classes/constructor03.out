8 .
-3 abc
